primeqa/qg/models/table_qg/sql_sampler.py

```diff
@@ -58,9 +58,15 @@ def sql_execution(self, where_clause, select_column, agg_op, table):
             [String]: [Answer after executing sql on the given table]
         """
         
+
         selected_cells = []
-        for row_id in where_clause['rows']:
-            selected_cells.append(table['rows'][row_id][select_column])
+        
```

Comment:
This part is not really related to moving the files. Should have been a separate PR. Maybe we should separate stuff like that in the future.

Reply:
@mihaela-bornea : Changes in sql_sampler should be okay because we included some more test cases to increase the QG test coverage after re_org. You can see the increase in qg test cases in /tests/qg/* commits.